src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.ClientCertificates.cs

```diff
@@ -208,5 +208,49 @@ public async Task AutomaticOrManual_DoesntFailRegardlessOfWhetherClientCertsAreA
                 }, new LoopbackServer.Options { UseSsl = true });
             }
         }
+
+        [ConditionalFact]
+        [PlatformSpecific(TestPlatforms.Android)]
+        public async Task Android_GetCertificateFromKeyStoreViaAlias()
+        {
+#if TARGETS_ANDROID
```

Comment:
Why do we need this compiler condition if the test is already marked as `PlatformSpecific`?

Reply:
The `AndroidKeyStoreHelper` is only available on android builds. I will change this test to be all wrapped in `#if` and remove the `PlatformSpecific`.